src/Tooltip.js

```diff
@@ -9,12 +9,19 @@ const propTypes = {
   disabled: PropTypes.bool,
   tether: PropTypes.object,
   toggle: PropTypes.func,
-  children: PropTypes.node
+  children: PropTypes.node,
+  delay: PropTypes.oneOfType([PropTypes.object, PropTypes.number])
```

Comment:
Yeah good idea, I'll use shape instead. And I can add a couple of tests to check that the `hide` or `show` defaults are picked up :+1: